src/apps/Console/ConsoleFeedback.js

```diff
@@ -243,15 +274,20 @@ export default function ConsoleFeedback({
               Agent #{index + 1}
             </Link>
           ))}
-        <p
-          css={`
-            ${textStyle('body2')}
-            margin: ${GU}px 0 ${GU}px 0;
-          `}
-        >
-          and then pass the parameters required for the agent execute function:
-          the target address, and the human-readable function call.
+      </>
+    )
+  } else if (
+    currentParsedCommand[0] === 'act' &&
+    currentParsedCommand.length >= 2
+  ) {
+    return (
+      <>
+        <p css={`${textStyle('body2')} margin: 0 0 ${GU}px 0;`}>
+          Pass the paremeters required for the agent execute function: the
```

Comment:
```suggestion
          Pass the paremeters required for the Agent's execute function: the
```

Would it also be possible to format these options as a list instead? That way we can add the name (e.g. `targetAddress`) as well as a short explanation.

Reply:
This sounds good!